src/overview/pricing/sponsored.md

```diff
@@ -0,0 +1,40 @@
+# Sponsored Community Sites
+
+Platform.sh provides sponsored hosting for Free Software projects and tech community events and organizations as part of our effort to support the Free Software community.
+
+## Eligibility
+
+* A non-trivial software project released under a Free and Open Source license (*GPL, BSD, MIT, Apache, etc.)
```

Comment:
```suggestion
* A non-trivial software project released under a Free and Open Source license (GPL, BSD, MIT, Apache, etc.)
```

Reply:
The * here is to indicate AGPL, LGPL, GPL, any version, etc.